Modix.Bot/Behaviors/PromotionPollBehavior/CachedPromoPoll.cs

```diff
@@ -0,0 +1,17 @@
+using Discord;
+using Modix.Data.Models.Promotions;
+
+namespace Modix.Bot.Behaviors
+{
+    public class CachedPromoPoll
+    {
+        public IUserMessage Message { get; set; }
+
+        public PromotionCampaignBrief Campaign { get; set; }
+
+        public int Approve { get; set; }
```

Comment:
Nitpick: `ApprovalCount` and `DisapprovalCount`. Definitely not `Disprove` though, that's a different word.